<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QDialog" name="OptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptionDialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vanishing line width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontal right line color:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical line color:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mass grammar id:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Silhouette color:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Silhouette width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>facade grammar id:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>window grammar id:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHorizontalLeftLineColor">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
background-color: blue;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontal left line color:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHorizontalRightLineColor">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonVerticalLineColor">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSilhouetteColor">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxVanishingLineWidth">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSilhouetteLineWidth">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMassGrammarId">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFacadeGrammarId">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxWindowGrammarId">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
